rocAL/rocAL/include/caffe2_meta_data_reader.h

```diff
@@ -40,6 +40,7 @@ public :
     void release(std::string image_name);
     void release() override;
     void print_map_contents();
+    void set_timestamps_bool() override { return; }
```

Comment:
Looks like this function is not doing anything? What is the use of this?

Reply:
set_timestamps_bool is used to set the variable _file_list_frame_num. If the _file_list_frame_num is set to true then the data from text file input will be interpreted as frame  number otherwise they will be interpreted as timestamps. set_timestamps_bool will be called in master_graph.cpp and it is introduced a virtual function in MetaDataReader hence we have defined it in all the meta data readers derived from the class class.